config/rbac/role.yaml

```diff
@@ -248,6 +248,22 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
+  - networking.k8s.io
+  resources:
+  - ingressclasses
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - networking.k8s.io
+  resources:
+  - ingressclasses/status
+  verbs:
+  - get
+  - patch
+  - update
```

Comment:
This resource _doesn't have a status_ so we should drop status permissions as they're superfluous.

Reply:
f4e6738d